internal/flink/command_statement_create.go

```diff
@@ -111,6 +116,28 @@ func (c *command) statementCreate(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	wait, err := cmd.Flags().GetBool("wait")
+	if err != nil {
+		return err
+	}
+	if wait {
+		err := retry.Retry(time.Second, time.Minute, func() error {
+			statement, err = client.GetStatement(environmentId, name, c.Context.LastOrgId)
+			if err != nil {
+				return err
+			}
+
+			if statement.Status.GetPhase() == "PENDING" {
+				return fmt.Errorf(`statement phase is "%s"`, statement.Status.GetPhase())
+			}
+
+			return nil
+		})
+		if err != nil {
+			return err
```

Comment:
Will this line return the `retry failed` error message? If so, maybe we should replace it with something more specific to this scenario.

Perhaps also adding a suggestion to run `describe` to see the current status if we timeout here.

Reply:
it'll say "retry failed: statement phase is PENDING"